packages/patternfly-3/patternfly-react-extensions/src/components/DiffView/DiffView.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { parseDiff, Diff, markCharacterEdits } from 'react-diff-view';
+import { formatLines, diffLines } from 'unidiff';
+
+const getDiff = (oldText, newText) => {
+  const diffText = formatLines(diffLines(oldText, newText), { context: 3 });
```

Comment:
from what I saw on `unidiff` repo, `context` defaults to 3 when nothing is specified,
could it be changed or set by the consumer? maybe you should add it to props.